server/src/services/map-updater.service.ts

```diff
@@ -40,6 +44,18 @@ export class MapUpdaterService {
     // tslint:disable-next-line:no-console
     console.log('reserves', reserves.features.length);
 
+    const conn = await this.databaseService.getConnection();
+    const mapDataCon = await conn.getRepository(MapData);
+
+    let mapData: MapData;
+
+
+    //save to table
+    mapData = {feature: 'reserve', properties: JSON.stringify(reserves.features)};
+    await mapDataCon.save(mapData);
```

Comment:
Maybe refactor by creating a service that saves to and reads from the database.

So rather than getting the database connection and saving manually, one could just inject a MapService (or something) and then do `this.mapService.saveFeature('reserve', reserves.features);`

Just to reduce repetition of code.

Reply:
This seems like a better option, will implement it.